cyences_app_for_splunk/default/savedsearches.conf

```diff
@@ -1806,6 +1806,195 @@ search = | savedsearch "Palo Alto Firewall - Network Compromise - DDoS Attack Pr
+action.cyences_notable_event_action.attacker_drilldown = `cs_fortigate` sourcetype="fortigate_utm" src=$row.src$
+action.cyences_send_email_action = 1
+action.cyences_notable_event_action.products = Fortigate
+
+[Fortigate Firewall - Network Compromise - Fortigate High System Alert]
+disabled = 1
+enableSched = 1
+alert.track = 1
+alert.severity = 4
+alert.suppress = 0
+counttype = number of events
+quantity = 0
+relation = greater than
+cron_schedule = 19,49 * * * *
+description = A Fortigate Firewall alerts for high or critical severity system events. Anything license-related is excluded from this alert as generally it's not that high severity. You can view them on the Fortigate dashboard.\
+\
+Data Collection - Fortigate firewalls system/events data (Fortinet Fortigate Add-on for Splunk).
+dispatch.earliest_time = -32m@m
+dispatch.latest_time = -2m@m
+display.general.type = statistics
+display.page.search.tab = statistics
+display.page.search.mode = fast
+request.ui_dispatch_app = cyences_app_for_splunk
+request.ui_dispatch_view = search
+search = `cs_fortigate` sourcetype="fortigate_event" severity IN ("critical", "high") NOT "*license*" \
+| eval cyences_severity = case(severity=="informational", "info", true(), severity) \
+| rename logdesc as description
+| table _time, subtype, signature, cyences_severity, action, description, dvc \
+| `cs_human_readable_time_format(_time, event_time)` \
+| `cs_fortigate_high_system_alerts_filter`
+action.cyences_notable_event_action = 1
+action.cyences_notable_event_action.param.filter_macro_name = cs_fortigate_high_system_alerts_filter
+action.cyences_notable_event_action.contributing_events = `cs_fortigate` sourcetype="fortigate_event" severity IN ("high" "critical") NOT "*license*"
+action.cyences_notable_event_action.system_compromised_search = | stats sum(count) as count by dvc
+action.cyences_notable_event_action.system_compromised_drilldown = `cs_fortigate` sourcetype="fortigate_event" severity IN ("high" "critical") NOT "*license*" dvc=$row.dvc$
+action.cyences_send_email_action = 1
+action.cyences_notable_event_action.products = Fortigate
+
+
+# Reports
+[Fortigate Firewall - Network Compromise - DDoS Attack Prevented]
+disabled = 1
+action.email.useNSSubject = 1
+alert.track = 0
+auto_summarize = 1
+auto_summarize.dispatch.earliest_time = -1mon@d
+description = Fortigate Firewall prevented DDoS attack events.\
+\
+Data Collection - Fortigate Networks firewalls threat data. (Fortinet Fortigate Add-on for Splunk)
+search = `cs_fortigate` sourcetype="fortigate_utm" \
+| stats count, latest(_time) as last_seen, dc(src_port) as dc_src_port, dc(dest_ip) as dc_dest_ip, values(signature) as signature, values(dvc) as dvc, values(srccountry) as src_location, dc(dest) as dc_dest, values(action) as action by src | sort - count \
+| `cs_fortigate_ddos_prevented_filter`
+action.cyences_notable_event_action.products = Fortigate
+
+[Fortigate Firewall - Network Compromise - Inbound Traffic from Blocked IPs]
```

Comment:
Can we generalize this alert with `Network Traffic` data-model?
- ensure it covers atleast Palo Alto and Fortigate logs?